android/java/org/chromium/chrome/browser/omnibox/BraveLocationBarCoordinator.java

```diff
@@ -39,7 +39,17 @@
 import java.util.function.BooleanSupplier;
 
 public class BraveLocationBarCoordinator extends LocationBarCoordinator {
+    /*
+     * To be deleted in bytecode. {@link LocationBarCoordinator#mLocationBarMediator} will be used
+     * instead.
```

Comment:
I think the comment should be something like ```
{@link LocationBarCoordinator#mLocationBarMediator} is private so we add a private `mLocationBarMediator` here so this code compiles and then remove it and make {@link LocationBarCoordinator#mLocationBarMediator} protected
```

Reply:
Updated comments